packages/neo-one-client-core/src/user/Dapi.ts

```diff
@@ -0,0 +1,396 @@
+   * @example compatibility: ['NEP-14', 'NEP-23','NEP-29']
+   */
+  readonly compatibility: ReadonlyArray<string>;
+  /**
+   * This object can contain any attributes specific to the `DapiProvider`, such as an app theme.
+   */
+  readonly extra: object;
+}
+
+/**
+ * Networks the `DapiProvider` has available to connect to, along with the default network.
+ */
+export interface DapiNetworks {
+  /**
+   * Networks the `DapiProvider` has available to connect to.
+   */
+  readonly networks: ReadonlyArray<NetworkType>;
+  /**
+   * Default network of the `DapiProvider`.
+   */
+  readonly defaultNetwork: NetworkType;
+}
+
+/**
+ * An account provided by a `DapiProvider`.
+ */
+export interface DapiAccount {
+  /**
+   * `AddressString` of the connected account
+   */
+  readonly address: AddressString;
+  /**
+   * A label the user has set to identify their wallet.
+   */
+  readonly label?: string;
+}
+
+export interface DapiPublicKey {
+  readonly address: AddressString;
+  readonly publicKey: PublicKeyString;
+}
+
+export interface BalanceRequest {
+  /**
+   * `AddressString`s to check balance(s)
+   */
+  readonly address: AddressString;
+  /**
+   * `Asset` ID or script hash to check balance.
+   */
+  readonly assets: Hash256String | ReadonlyArray<Hash256String>;
+  /**
+   * Fetches to UTXO data for NEO and/or GAS if attribute is 'true'
+   */
+  readonly fetchUTXO?: boolean;
+}
+
+export interface Balance {
+  readonly assetID: Hash256String;
+  readonly symbol: string;
+  readonly amount: string;
+}
+
+export interface BalanceResults {
+  readonly [address: string]: ReadonlyArray<Balance>;
+}
+
+/**
+ * Options for requesting balance
+ */
+export interface GetBalanceArgs {
+  /**
+   *
+   */
+  readonly params: BalanceRequest | ReadonlyArray<BalanceRequest>;
+  /**
+   * Network to submit this request to. If omitted, will default to network the `DapiProvider` is currently set to.
+   */
+  readonly network?: NetworkType;
+}
+
+export interface GetStorageArgs {
+  /**
+   * script hash of the smart contract to invoke a read on
+   */
+  readonly scriptHash: string;
+  /**
+   * key of the storage value to retrieve from the contract
+   */
+  readonly key: string;
+  /**
+   * Network to submit this request to. If omitted, will default to network the wallet is currently set to.
+   */
+  readonly network?: string;
+}
+
+export interface InvokeReadArgs {
+  /**
+   * script hash of the smart contract to invoke a read on
+   */
+  readonly scriptHash: string;
+  /**
+   * operation on the smart contract to call
+   */
+  readonly operation: string;
+  /**
+   * any input arguments for the operation
+   */
+  readonly args: ReadonlyArray<Argument>;
+  /**
+   * Network to submit this request to. If omitted, will default to network the wallet is currently set to.
+   */
+  readonly network?: string;
+}
+
+export interface BlockTransactionDetails {
```

Comment:
If we don't use this anywhere I would just leave this commented out so we don't use something typed with `any`. Either that, or add the full types in.